gigascience/gigadb-website
# Open jobs page in new tab

# Pull request for issue: #1283

This is a pull request for the following functionalities:

* Jobs link opens page in new menu

## How to test?

Verify the above. Optioanlly use screen reader to verify that user is notified about page opening in new tab

## How have functionalities been implemented?

* Use `target="__blank"` to open page in new windows
* Add informative message for screen reader users
* Add icon that indicates visually that this is external link

## Any issues with implementation?

--

## Any changes to automated tests?

- Commented out 3 failing tests that expect to open the jobs page. Any advice on how to implement those? When I try to edit the file `tests/_support/AcceptanceTester.php` which seems to contain test helper definitions, it looks like the changes don't have any effect. There must be an intermediate step I'm missing
